keras_hub/src/models/image_text_detector.py

```diff
@@ -0,0 +1,277 @@
+import keras
+import numpy as np
+
+from keras_hub.src.api_export import keras_hub_export
+from keras_hub.src.models.task import Task
+
+
+@keras_hub_export("keras_hub.models.ImageTextDetector")
+class ImageTextDetector(Task):
+    """Base class for all text detection tasks.
+
+    `ImageTextDetector` tasks wrap a `keras_hub.models.Task` and
+    a `keras_hub.models.Preprocessor` to create a model that can be used for
+    image segmentation.
+
+    All `ImageTextDetector` tasks include a `from_preset()` constructor which
+    can be used to load a pre-trained config and weights.
+
+    Args:
+        detection_thresh: The value for thresholding predicted mask outputs.
```

Comment:
Are these general concepts that will apply to other models? We should only leave common stuff here. If this would not apply to a non-diff bin model, we should leave it out.